Teensy/src/gpio/gpio.cpp

```diff
@@ -11,59 +11,59 @@
 #include <Arduino.h>
 #include "gpio.h"
 
-static int32_t gpio_getWheelSpeed();
-static uint32_t gpio_getEngineRevs();
-static int32_t gpio_getDamperPosition();
-static int32_t gpio_getGearPosition();
-static uint32_t gpio_getSteeringWheelPosition();
-static int32_t gpio_getStrain();
+static daq_WheelSpeed_t gpio_getWheelSpeed();
+static daq_EngineRev_t gpio_getEngineRevs();
+static daq_DamperPos_t gpio_getDamperPosition();
+static daq_GearPos_t gpio_getGearPosition();
+static daq_SteeringWhlPos_t gpio_getSteeringWheelPosition();
+static daq_Strain_t gpio_getStrain();
 static GyroData_s gpio_getGyro();
-static uint32_t gpio_getVBat();
-static uint32_t gpio_getThrottlePosition();
-static uint32_t gpio_getFuelPressure();
+static daq_BatteryV_t gpio_getVBat();
+static daq_ThrottlePos_t gpio_getThrottlePosition();
+static daq_FuelPressure_t gpio_getFuelPressure();
 
 bool updateSensorInfo(sensorData_s *pSensorData)
 {
-    pSensorData->wheel_speed = gpio_getWheelSpeed();
-    pSensorData->engine_revs = gpio_getEngineRevs();
-    pSensorData->damper_position = gpio_getDamperPosition();
-    pSensorData->gear_position = gpio_getGearPosition();
-    pSensorData->steering_wheel_position = gpio_getSteeringWheelPosition();
+    pSensorData->wheelSpeed_mph = gpio_getWheelSpeed();
```

Comment:
Wondering if m/s would be a more useful unit. Undecided on this though!

Reply:
I guess it doesn't matter too much, the server can just convert it to whatever unit the user wants?